rllib/train.py

```diff
@@ -217,59 +112,60 @@ def patch_path(path):
 
             exp["config"]["input"] = patch_path(input_)
 
-        if not exp.get("run"):
-            parser.error("the following arguments are required: --run")
         if not exp.get("env") and not exp.get("config", {}).get("env"):
-            parser.error("the following arguments are required: --env")
-
-        if args.torch:
+            raise ValueError(
+                "You either need to provide an --env argument or pass"
+                "an `env` key with a valid environment to your `config`"
+                "argument."
+            )
+        if torch:
             deprecation_warning("--torch", "--framework=torch")
             exp["config"]["framework"] = "torch"
-        elif args.eager:
+        elif eager:
             deprecation_warning("--eager", "--framework=[tf2|tfe]")
             exp["config"]["framework"] = "tfe"
```

Comment:
can you change this to "tf2"?

"tfe" should also be deprecated, soon (same behavior as "tf2").